README.md

```diff
@@ -0,0 +1,81 @@
+Dell PowerConnect Switch
+========================
+
+This role configures Dell PowerConnect switches using the `expect` Ansible
+module.
+
+Requirements
+------------
+
+The switches should be configured to allow SSH access.
+
```

Comment:
Given there are local_action pip installations, perhaps flag that up here and advise the use of a virtualenv?

Reply:
I think it would require some changes to use a virtualenv - currently the package is installed to the system packages.